dashboard/scripts/archive/generate_trusted_proxies

```diff
@@ -29,7 +29,7 @@ cloudfront_ips = response['prefixes'].
                  map {|ip_range| ip_range['ip_prefix']}
 
 output = {
-  comment: "Generated by #{__FILE__} at #{Time.now} from #{IP_RANGES_URL}",
+  comment: "Generated by #{__FILE__} from #{IP_RANGES_URL}",
   syncToken: response['syncToken'],
   createDate: response['createDate'],
```

Comment:
I noticed there's quite a difference in update frequency between the [CloudFront-only portion of this list](https://github.com/joetek/aws-ip-ranges-json/commits/master/ip-ranges-cloudfront.json) and the [full list](https://github.com/joetek/aws-ip-ranges-json/commits/master/ip-ranges.json), so I'd suggest removing the `syncToken` and `createDate` (both of which track updates the full list) from this output hash in order to reduce the frequency of git-committed updates to just the CloudFront data.